src/entity.js

```diff
@@ -158,8 +158,8 @@ entity.isDsGeoPoint = isDsGeoPoint;
  * Build a Datastore Key object.
  *
  * @class
- * @param {object|string|array} Configuration object.
- * @param {string|array} options.path  Key path.
+ * @param {object} options Configuration object.
+ * @param {array} options.path Key path.
```

Comment:
Thanks for catching these mis-types. It looks like the rest of the docs markup doesn't use hyphens, so I took them out.

Reply:
Great! I'll keep that in mind for any future doc changes 👍